eve/main.yml

```diff
@@ -0,0 +1,91 @@
+version: "0.2"
+
+branches:
+  default:
+    stage: "pre-merge"
+
+stages:
+  pre-merge:
+    worker: &pod
+      type: kube_pod
+      path: eve/workers/zenko.yaml
+    steps:
+    - TriggerStages:
+        name: trigger all the tests
+        stage_names:
+        - single-node
+
+  single-node:
+    worker: &openstack
+      type: openstack
+      image: CentOS 7 (PVHVM)
+      flavor: general1-4
+      #path: eve/workers/centos7
+    steps:
+    - Git: &git_pull
+        name: git pull
+        repourl: "%(prop:git_reference)s"
+        mode: full
+        method: clobber
+        retryFetch: true
+        haltOnFailure: true
+    - ShellCommand:
+        name: 'create loopback blockdevice and configure VM'
+        haltOnFailure: true
+        command: |-
+          sudo ./eve/single_node/single_node.sh
+    - ShellCommand:
+        name: 'Create virtual env'
+        haltOnFailure: true
+        command: |-
+          make shell
+    - ShellCommand:
+        name: 'Run the install with ansible'
+        haltOnFailure: true
+        command: >-
+          ./.shell-env/metal-k8s/bin/ansible-playbook
+          -i eve/single_node/
+          metal-k8s.yml --skip elasticsearch
+        env:
+          ANSIBLE_FORCE_COLOR: 'true'
+    - ShellCommand:
+        name: 'Run sample kubectl command'
+        haltOnFailure: true
+        command: /usr/local/bin/kubectl get nodes
+        env:
+          KUBECONFIG: inventories/single-node-ci/artifacts/admin.conf
+    - ShellCommand:
+        name: 'Test storage'
+        haltOnFailure: true
+        command: |-
+            /usr/local/bin/kubectl get pv
+            /usr/local/bin/kubectl apply -f tests/storage/test_pv.yml
+            #retry kubectl get pods test-volume -o jsonpath={.status.phase} | grep -v -E '(Failed|Completed)' ]
+            sleep 10
+            /usr/local/bin/kubectl delete pod test-pv
```

Comment:
We're not really checking somehow whether the test ran inside the `Pod` succeeded?

Reply:
Not yet indeed